widgetastic_manageiq/__init__.py

```diff
@@ -5513,6 +5523,50 @@ def get_progress_percent(self, plan_name):
         ]
         return dict(list(zip(desc, val)))
 
+    def schedule_cutover(self, plan_name, cancel=False, immediate=False, after_mins=5):
+        try:
+            el = self._get_card_element(plan_name)
+            cutover_button = self.browser.element(self.ITEM_CUTOVER_BUTTON_LOCATOR, parent=el)
+            wait_for(
+                lambda: cutover_button.is_enabled(),
+                delay=5,
+                num_sec=120,
+                message="waiting for cutover button to be enabled.",
+            )
+            cutover_button.click()
+            if immediate:
+                self.root_browser.click(self.IMMEDIATE_BTN)
```

Comment:
@mnadeem92 : Isn't the "schedule immediately" button selected by default on the UI ?
If yes then we don't have to click on it we can just check : 

    if not immediate: 
              cutover code 

So if `immediate=True` then it will skip the cutover code and just click on schedule button.

Reply:
@sshveta  For cutover, it by default select the schedule radio button and not the immediate one, which is why i need to select immediate button first here.